scipy/interpolate/fitpack2.py

```diff
@@ -543,7 +543,8 @@ class InterpolatedUnivariateSpline(UnivariateSpline):
         not in the interval defined by the knot sequence.
 
         * if ext=0 or 'extrapolate', return the extrapolated value.
-        * if ext=1 or 'zeros', return 0
+        * if ext=1 all(diff(x) > 0.0):
++            raise ValueEror 'zeros', return 0
```

Comment:
This is not valid [restructuredtext](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) markup. Also not the strange `+` at the start of the line.

Reply:
Thanks so much! I didn't even notice that.